machine_learning_hep/analysis/analyzer_jet.py

```diff
@@ -24,7 +24,10 @@
 # pylint: disable=import-error, no-name-in-module
 import uproot
 from ROOT import TFile, TH1F, TH2F, TCanvas, TLatex, TGraphAsymmErrors, TLine, TGaxis
-from ROOT import AliHFInvMassFitter, AliVertexingHFUtils
+try:
+    from ROOT import AliHFInvMassFitter, AliVertexingHFUtils
+except ImportError:
+    print("\n\033[1mFutureWarning: not in AliPhysics environment. Analyzer will crash!\033[0m\n")
```

Comment:
This should maybe also  not be a "FutureWarning" cause that would be something which refers to something coming/changing in the future, e.g. introducing changes to the API. 
How about just warn and put a `sys.exit(1)`?

Reply:
As discussed on Mattermost, `sys.exit(1)` is not the way to go here. Idea now is to introduce a decorator (`@requiresaliphysics`). Issue to be opened